### ISSUE TITLE
Missing release information in Dependabot PRs for crates other than `libcnb`

### ISSUE THREAD

**Issue by edmorley** [Malax/libcnb.rs#327]
Currently Dependabot PRs for crates other than `libcnb` have no release/changelog information listed in the PR description:

eg:

<img width="756" alt="libcnb-test Dependabot PR screenshot" src="https://user-images.githubusercontent.com/501702/154272687-acb0da11-beda-4898-842a-233c9a354791.png">

In addition, the "compare view" link shown in that screenshot links to https://github.com/Malax/libcnb.rs/commits/v0.1.1 which is invalid, since (a) there is no such tag, (b) even if there were, it would be a tag for `libcnb` not `libcnb-test`, since we don't currently tag for other crates.

I think we should:
1. Start prefixing our git tags with the crate name (eg `libcnb/vX.Y.Z` not `vX.Y.Z`)
2. Start creating git tags + GitHub releases for the other crates too
3. For bonus points, paste the actual changelog content into the GitHub release, copied from the real changelog markdown.

(1)+(2) will resolve the missing Dependabot metadata, since it favours releases/tags containing the dependency name over the bare version names:
https://github.com/dependabot/dependabot-core/blob/c234a3c87266216d671cb38976ee0049eff81215/common/lib/dependabot/metadata_finders/base/release_finder.rb#L44-L47

Suggested naming scheme:
- Git tag: `<CRATE_NAME>/vX.Y.Z`
- GitHub release title: `<CRATE_NAME> vX.Y.Z` (ie: same as Git tag but with the `/` replaced with a space)

**Comment by edmorley** [comment]
> Start prefixing our git tags with the crate name (eg `libcnb/vX.Y.Z` not `vX.Y.Z`)

I've renamed all tags to match this naming format:
`<CRATE_NAME>/vX.Y.Z`

...and created tags for all other crates. See:
https://github.com/Malax/libcnb.rs/tags

Changing the tag name reset the two GitHub releases I'd previously created back to "draft", so I ended up deleting them (the release names didn't match the new naming scheme anyway) since re-publishing them would reset the release date to now, rather than the actual release date.

Moving forwards we should create releases using this naming scheme:
`<CRATE_NAME> vX.Y.Z`

I've filed #335 for documenting the release process including the above.
